app/components/Button.tsx

```diff
@@ -13,27 +15,32 @@ export const Button: React.FC<ButtonProps> = ({
   label,
   varient = 'primary',
   disabled = false,
-  handleClick,
+  type = 'submit',
+  style,
+  handleClick = () => {},
 }) => (
   <button
     className={`
                 flex items-center justify-center border rounded-lg transition 
                w-full
                 ${
                   varient === 'primary'
-                    ? 'border-transparent bg-blue-600 text-white hover:bg-blue-800'
+                    ? `border-transparent ${
+                        disabled ? 'bg-stone-500' : 'bg-stone-900'
+                      }   text-white hover:bg-stone-1100`
                     : 'bg-white border-gray-400 text-gray-900 hover:bg-gray-300'
                 } 
                 ${
                   size === 'small'
-                    ? 'py-1 px-2 text-sm gap-1'
+                    ? 'py-1 px-2 my-2 text-sm gap-1'
                     : size === 'medium'
-                    ? 'px-4 py-2 gap-2'
-                    : 'px-4 py-2 text-lg gap-3'
+                    ? `${style} px-4 py-2 gap-2`
+                    : `${style}  px-4 py-2 text-lg gap-3`
```

Comment:
keep the `style` variable at the end so that we can override existing classes if needed later